test.py

```diff
@@ -487,6 +481,10 @@ class PythonTestSuite(TestSuite):
     def __init__(self, path, cfg: dict, options: argparse.Namespace, mode: str) -> None:
         super().__init__(path, cfg, options, mode)
         self.scylla_exe = path_to(self.mode, "scylla")
+        if not os.access(self.scylla_exe, os.F_OK):
+            raise FileNotFoundError(f"{self.scylla_exe} does not exist.")
+        if not os.access(self.scylla_exe, os.X_OK):
+            raise PermissionError(f"{self.scylla_exe} is not executable.")
```

Comment:
Would that prevent running a unit tests without building scylla?

Reply:
yes, but this is the existing behaviour.

Reply:
It doesn't make much sense to me but if it's the existing behavior we need not change it in this PR.

Reply:
https://github.com/scylladb/scylladb/issues/22168